### PR TITLE
draft release notes for 1.16.3

### PR DESCRIPTION
**Pull request by AtnNn**
@mglukhovsky @danielmewes Could you please review?

### CODE REVIEW

File: NOTES.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
+# Release 1.16.3 (Stand By Me)
+
+Released on 2015-03-26
+
+Bug fix update.
+
+* Fixed a bug that could cause a crash when reading from a secondary index in some rare circumstances (#3976)
+* Fixed `rethinkdb export` compatibility between Python 2 and Python 3 (#3911)
+* Include the remote port number in the heartbeat timeout message (#2891)
```
Comment (mglukhovsky) [inline NOTES.md]: I suggest:

> Heartbeat timeout messages now include the remote port number.

**Comment by mglukhovsky** [comment]
Left one note, otherwise looks good to me!

**Comment by danielmewes** [comment]
I'd also like to include https://github.com/rethinkdb/rethinkdb/issues/3954 in this.
Suggestion:
"Fixed a bug that could cause a connection to hang indefinitely on OS X (#3954)"

**Comment by AtnNn** [comment]
Thanks. I updated the notes.

**Comment by danielmewes** [comment]
Looks good. Thank you @AtnNn

**Pull request closed without merging**